Dockerfile

```diff
@@ -11,12 +15,14 @@ RUN tar -xf /tmp/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz -C /tmp \
 
 RUN apk add --update git \
     && apk upgrade \
-    && apk add --no-cache ca-certificates
+    && apk add --no-cache ca-certificates \
+    && apk add sudo
```

Comment:
Please add `--no-cache` here as well to avoid saving a copy of the `apk` index of packages within the compiled container.

Reply:
OK